test/grouping.jl

```diff
@@ -2334,6 +2360,13 @@ end
         @test parent(gdf2).y ≅ df.y
         @test parent(gdf2).g === df.g
 
+        # Test that nthreads argument is accepted
```

Comment:
should be removed now - right?

Reply:
Yes. I'm not sure how tests passed locally. There's something weird going on. :-/